### ISSUE TITLE
Manual Instructions for Installing Sublime Package Control

### ISSUE THREAD

**Issue by entawngledcipher** [wbond/package_control#253]
I am using Ubuntu 12.04. 

Just tried this mentioned on the website documentation :

Manual Instructions

If for some reason the console installation instructions do not work for you (such as having a proxy on your network), perform the following steps to manually install Package Control:

1>Click the Preferences > Browse Packages… menu entry
2>Browse up a folder and then into the Installed Packages folder
3>Download Package Control.sublime-package and copy it into the Installed Packages directory
4>Restart Sublime Text

However, when I restarted Sublime it gave me an error saying, Cannot Start Package Controller.
Basically it was not able to install the Package Controller.

Instead of following step 3 what I did is :

3> Extract the files from Package Control.sublime-package.zip into a folder.
Create a new folder under the Packages folder named Package Control and copy all the extracted files.

Vola!!!!1 It worked....

Let me know if this is cool or will give me issues in the future :)

**Comment by bjorn** [comment]
I was having a similar issue on Windows 7 with Sublime Text 3103 (portable package). After following the manual install instructions, Package Control was simply not there after restarting Sublime Text (I didn't get any error message either).

And after following these suggestions, namely extracting the .zip file and putting it in "Packages" instead of "Installed Packages", it worked.

**Comment by kapjames** [comment]
I was having the exact issue on Windows 7. Tried Sublime Text 2 and 3, several new installations, checked ignored packages in preferences, but Package Manager seemed to simply not existed. Followed the steps bjorn tried and worked perfectly.

**Comment by audiolion** [comment]
had the same issue, with sublime text 3 portable and windows 7, no error message, after restarting it just wasn't there, followed bjorn/entawngledcipher instructions and it worked

- located install location of sublime by going to Preferences -> Browse Packages
- make a new folder in ./Packages called Package Control
- unzip into that folder
